### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by KojoSwiss**
I added the login, Log out functionality to the navbar and also the flash alerts to the application.html.erb page.
NB: ⚠️
There is an issue with the typed.js which prevents the Log out function. The log out only works when parts of the typed.js are commented out. Please see the screenshot below.
Screenshot 2020-11-23 at 18 32 15

### CODE REVIEW

**Comment by Simon-Busch** [comment]
Hey Ben,

I don't udnerstand why it would create a conflict ?

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,9 +12,8 @@
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
```
Comment (Simon-Busch) [inline app/views/layouts/application.html.erb:16]: Why did you delete this ??

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -12,9 +12,8 @@
...
     <%= render './components/navbar' %>
+    <%= render 'shared/flashes' %>
```
Comment (Simon-Busch) [inline app/views/layouts/application.html.erb:16]: Can you please explain what this is ?

**Comment by KojoSwiss** [comment]
It was because of where the navbar was located. I don't know why but when I moved it into the shared folder it works. PLEASE DO NOT MERGE THIS PULL REQUEST FOR NOW

**Review by Simon-Busch — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (KojoSwiss) [reply app/views/layouts/application.html.erb:16]: For best practise, it is better to have the navbar in the shared folder since it is going to be used by all the pages  :)

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (KojoSwiss) [reply app/views/layouts/application.html.erb:16]: I moved then to the shared folder and just rendered on the pages where they are used
